CHANGES.rst

```diff
@@ -1,3 +1,73 @@
+23.2.0 (To be determined)
+=========================
+New feature release in the 23.2.x series.
+
+This release wraps up a significant refactor of fMRIPrep. The main new features
+can be used with the ``--level`` and ``--derivatives`` flags.
+
+The ``--level`` flag can take the arguments ``minimal``, ``resampling`` or
+``full``. The default is ``full``, which should produce nearly the same results
+as previous versions. ``minimal`` will produce only the minimum necessary to
+deterministically generate the remaining derivatives. ``resampling`` will produce
+some additional derivatives, intended to simplify resampling with other tools.
+
+The ``--derivatives`` flag takes arguments of the form ``name=/path/to/dir``.
+If provided, fMRIPrep will read the specified directories for pre-computed
+derivatives. If a derivative is found, it will be used instead of computing
+it from scratch. If a derivative is not found, fMRIPrep will compute it and
+proceed as usual.
+
+Taken together, these features can allow a dataset provider to run a minimal
+fMRIPrep run, targeting many output spaces, while a user can then run a
+``--derivatives`` run to generate additional derivatives in only the output
+spaces they need. Another use case is to provide an precomputed derivative
+to override the default fMRIPrep behavior, enabling easier workarounds for
+bugs or experimentation with alternatives.
+
```

Comment:
Not sure how accurate this is, but it feels like mentioning MSMSulc is worthwhile.

```suggestion

A second prominent feature included within this release is the integration
of MSMSulc for surface-to-surface registration. Along with this integration,
CIFTI-2 and other related output options have been restored.

```